src/main/resources/interface-specification.yml

```diff
@@ -4011,6 +4011,86 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /purposes/{purposeId}/versions/{versionId}/suspend:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+      - name: purposeId
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+      - name: versionId
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      tags:
+        - purposes
+      summary: Suspend Purpose Version
+      operationId: suspendPurposeVersion
+      responses:
+        '200':
+          description: Purpose Version Suspended
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CreatedResource'
```

Comment:
@Carminepo2 in these endpoints what do you need as return value?
- purposeId
- versionId
- both purposeId and versionId

Reply:
Both would be great!

Reply:
@AsterITA I would suggest a model named `PurposeVersionResource` which contains the two ids.
This model will be used in all other endpoints where we create/update a purpose

Reply:
@galales in this case should we reuse [this schema](https://github.com/pagopa/interop-be-backend-for-frontend/blob/cd2cb1340d0b243c63d1ad81e2d097e78fb229f2/src/main/resources/interface-specification.yml#L5926) or create another one?

Reply:
Despite preferring avoiding duplication, I would say to create a new model.
Because a generic model would have also generic fields names that could lead to ambiguities.
This way we can name fields `purposeId` and `versionId` that are straightforward.